Recipes/Along_Isobath_Average.ipynb

```diff
@@ -33,12 +33,18 @@
    "metadata": {},
```

Comment:
<p>Adding a %%time on this cell would be handy - it's very intensive, so giving the user an idea of how long it's going to take would be good.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/499/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>